yo/main.py

```diff
@@ -291,8 +291,8 @@ def load_config(config_file: str = CONFIG_FILE) -> FullYoConfig:
 
 
 def ssh_args(
-    ctx: YoCtx,
-    interactive: bool,
+        ctx: YoCtx,
+        interactive: bool,
```

Comment:
Please do not reformat unrelated code.

To add to this, please see the [Development](https://oracle.github.io/yo/development.html) information in the documentation. In particular, the command `make development` should install [pre-commit](https://pre-commit.com) hooks which will enforce coding style requirements. The files already adhere to the "black" code style. If you had the hooks installed, it would have automatically fixed the code style.

This applies to all the whitespace changes shown here. The diff should only consist of changes necessary to add the new command.

Reply:
I have made the changes as per the guidelines and suggestions, and regarding the reformatting, I am manually not changing the formatting from my side, this time I even installed the pre-commit hooks so that while committing the changes it enforces proper formatting as per the formatting regulations, kindly let me know if its done.